### PR TITLE
feat: add command login mode

### LINKED ISSUE #10: [Feature Request] Support for Hot-Seat LAN
This is an awesome mod, I see it being helpful in a closed LAN to keep people from login in as another player, which has 2 negative effects. One, it kicks the other person out, and second, it gives the person access to the other persons player and inventory, etc...

### ISSUE
Have multiple players, and multiple PCs, players randomly use different PCs.

For a LAN where everyone logs into the game from the same PCs every time this mod works great as is. But when you have a cyber cafe, or in my case a house with a bunch of PCs with players not always using the same PC, it makes it real difficult to delete the login files, or unregistered the player from the server.

### REQUEST
It would be nice to have a config option in the server and client that doesn't store the login persistently, but store the password / player name only on the server encrypted and requires login username and password every time from the client side that would allow multiple different players to login at different times from the same PC or the same player name to log into the server from different PCs.  Both options would need to be implemented for this to work.

### PR DESCRIPTION
**Pull request by SeraphJACK**
Add command login mode.

BREAKING CHANGE: password stored before this version will be invalid if upgraded mod version.
Closes #10

### CODE REVIEW

**Comment by SeraphJACK** [comment]
Not tested yet, I need someone to help me test this.

**Comment by SeraphJACK** [comment]
Ready for test and review now.

File: src/main/java/top/seraphjack/simplelogin/client/command/CommandLogin.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    @Override
+    public String getUsage(ICommandSender sender) {
+        return "/login password";
+    }
+
+    @Override
+    public void execute(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException {
+        if (args.length != 1) {
```
Comment (Yesterday17) [inline src/main/java/top/seraphjack/simplelogin/client/command/CommandLogin.java:29]: Password with space?

File: src/main/java/top/seraphjack/simplelogin/client/command/CommandRegister.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    @Override
+    public String getUsage(ICommandSender sender) {
+        return "/register <Password> <Password>";
+    }
+
+    @Override
+    public void execute(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException {
+        if (args.length != 2) {
```
Comment (Yesterday17) [inline src/main/java/top/seraphjack/simplelogin/client/command/CommandRegister.java:29]: Same as previous one

**Review by Yesterday17 — CHANGES_REQUESTED** [review]
rua

File: src/main/java/top/seraphjack/simplelogin/client/command/CommandLogin.java (hunk 1 above)
Comment (SeraphJACK) [reply src/main/java/top/seraphjack/simplelogin/client/command/CommandLogin.java:29]: no, no space

File: src/main/java/top/seraphjack/simplelogin/client/command/CommandRegister.java (hunk 2 above)
Comment (SeraphJACK) [reply src/main/java/top/seraphjack/simplelogin/client/command/CommandRegister.java:29]: no, or how to split args when registering?

Comment (Yesterday17) [reply src/main/java/top/seraphjack/simplelogin/client/command/CommandRegister.java:29]: The same as how wpmcmc handles space(

**Review by Yesterday17 — APPROVED** [review]

**Pull request merged**